src/block/horizontal-scroller/edit.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Internal dependencies
+ */
+import BlockStyles from './style'
+
+/**
+ * External dependencies
+ */
+import { version as VERSION, i18n } from 'stackable'
+import classnames from 'classnames'
+import {
+	 ColumnInnerBlocks,
+	 GroupPlaceholder,
+	 InspectorTabs,
+	 AdvancedRangeControl,
+	 InspectorStyleControls,
+	 PanelAdvancedSettings,
+	 AdvancedSelectControl,
+} from '~stackable/components'
+
+import {
+	 BlockDiv,
+	 useGeneratedCss,
+	 MarginBottom,
+	 Alignment,
+	 getAlignmentClasses,
+	 Advanced,
+	 CustomCSS,
+	 Responsive,
+	 CustomAttributes,
+	 EffectsAnimations,
+	 ConditionalDisplay,
+	 Transform,
+	 ColumnsControl,
+	 getContentAlignmentClasses,
+} from '~stackable/block-components'
+import { useBlockContext } from '~stackable/hooks'
+import {
+	 withBlockAttributeContext,
+	 withBlockWrapperIsHovered,
+	 withQueryLoopContext,
+} from '~stackable/higher-order'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+
+const Edit = props => {
+	 const {
+		 className,
+		 clientId,
+		 isSelected,
+	 } = props
+
+	 useGeneratedCss( props.attributes )
+
+	 const blockAlignmentClass = getAlignmentClasses( props.attributes )
+	 const { hasInnerBlocks } = useBlockContext()
+	 const [ columnProviderValue, columnTooltipClass ] = ColumnInnerBlocks.useContext()
+
+	 const blockClassNames = classnames( [
+		 className,
+		 'stk-block-horizontal-scroller editor',
```

Comment:
I think you added an `editor` class here so you can check whether the scroller is in the editor or in the frontend. From what I can see, I think you're using it for 2 things:

1. Frontend js script for the horizontal scroller in `frontend-horizontal-scroller.js`
2. Styling for the frontend only

I believe you can remove the `editor` class here and do this instead:

1. in `frontend-horizontal-scroller.js`, remove the reference to `.editor`, this script is only ran in the frontend anyway
2. In `horizontal-scroller/style.scss`, remove the `:not(.editor)`, instead, add an `editor.scss` and override the styles you don't want in the editor, e.g. display grid

Reply:
@bfintal this script is also being run on the editor, this is so user can check the actual scroll snapping behavior even when on the editor.

Reply:
@bfintal for this one, I'm targeting a different element between backend vs frontend, this is to avoid applying the styles in the editor for the class `.stk-block-content` as those styles are not needed on the editor.

for the frontend script, will remove the editor selector